### PR TITLE
Add validation

### PR DESCRIPTION
**Pull request by rogachevsn55**
add validation to DID param, add Service element to DDO, change status code for DIDDuplicateException

### CODE REVIEW

File: src/test/resurces/ddo.json [hunk 1]
```diff
@@ -25,10 +25,10 @@
 
   "created": "2002-10-10T17:00:00Z",
```
Comment (Warchant) [inline src/test/resurces/ddo.json]: check the path for this file, you miss letter:
src/test/res(**o**)urces/ddo.json

File: src/test/resurces/ddo.json [hunk 2]
```diff
@@ -25,10 +25,10 @@
...
 
-  "proof": {
+  "proof": [{
     "type": "Ed25519Signature2018",
     "created": "2002-10-10T17:00:00Z",
     "creator": "did:sora:ed:H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV#keys-3",
     "signatureValue": "QNB13Y7Q9...1tzjn4w=="
-  }
+  }]
 }
```
Comment (Warchant) [inline src/test/resurces/ddo.json]: setup your IDE to add a line ending at the end of each file:
- [x] Ensure line feed at the end on Save

File: src/main/java/jp/co/soramitsu/sora/didresolver/DTO/Service.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+
+/**
+ * @author sbt-rogachev-sn
```
Comment (Warchant) [inline src/main/java/jp/co/soramitsu/sora/didresolver/DTO/Service.java]: is this comment necessary? 
IMO it is better to remove autogenerated comments like this

File: src/test/resurces/ddo.json (hunk 1 above)
Comment (rogachevsn55) [reply src/test/resurces/ddo.json]: done

File: src/test/resurces/ddo.json (hunk 2 above)
Comment (rogachevsn55) [reply src/test/resurces/ddo.json]: done

File: src/main/java/jp/co/soramitsu/sora/didresolver/DTO/Service.java (hunk 3 above)
Comment (rogachevsn55) [reply src/main/java/jp/co/soramitsu/sora/didresolver/DTO/Service.java]: remove comments

**Pull request merged**